model_compression_toolkit/gptq/pytorch/quantizer/soft_rounding/soft_quantizer_reg.py

```diff
@@ -40,32 +40,36 @@ def __init__(self, beta_scheduler: Callable[[int], float]):
 
         self.count_iter = 0
 
-    def __call__(self, model: nn.Module, entropy_reg: float):
+    def __call__(self, model: nn.Module, entropy_reg: float, layer_weights: torch.Tensor = None):
```

Comment:
If I'm not mistaken, the default case (where the weighting is all ones) is taking care outside of this function. If this is the case, maybe it is may be better to remove the default value of layer_weights?

Reply:
You are not mistaken. I will remove it for now due to missing testing/coverage, but in principle I don't see a connection between the two. I think in this case it makes sense to have a default behavior, the fact that it was eventually more convenient to pass explicit weights in the specific use case should not really affect this.